### PR TITLE
branchy branch

### PR DESCRIPTION
**Pull request by meowbot24**
mao

### CODE REVIEW

File: rockpaperscissors.js [hunk 1]
```diff
@@ -92,5 +92,5 @@ while (userChoice === computerChoice) {
 	compare(userChoice, computerChoice);
 };
 
-// next to do: get rid of repetitive code in last while loop
+// next to do: get rid of repetitive code in last while loop mao mao
```
Comment (scottybarr) [inline rockpaperscissors.js:95]: Mao! 😸 
Merge it!

**Comment by scottybarr** [comment]
@meowbot24 Press `Delete Branch` after you merge one :)

**Comment by meowbot24** [comment]
@scottybarr Okie done! Sorry I'm such a n00b 😳

**Comment by scottybarr** [comment]
@meowbot24 It's cool! Just good practice going forward! 😸

**Comment by meowbot24** [comment]
@scottybarr Thanks mao! Okay so what now? 😹

**Pull request merged**